table/block_based_table_reader.cc

```diff
@@ -2880,25 +2882,28 @@ void BlockBasedTable::MultiGet(const ReadOptions& read_options,
       bool matched = false;  // if such user key matched a key in SST
       bool done = false;
       for (iiter->Seek(key); iiter->Valid() && !done; iiter->Next()) {
-        DataBlockIter biter;
-        NewDataBlockIterator<DataBlockIter>(
-            rep_, read_options, iiter->value(), &biter, false,
-            true /* key_includes_seq */, get_context);
+        if (iiter->value().offset() != bhandle.offset()) {
```

Comment:
What's the initial value of `bhandle.offset()` before it is initialized? If it is 0, how if iiter->value().offset() is also 0?

Reply:
Yes, that's a good point. It looks like the default constructor initializes ```offset_``` to 0. To make it more robust, I'll check both offset and size and initialize ```bhandle``` to ```kNullBlockHandle```.